src/Checkbox/__tests__/__snapshots__/Checkbox.test.js.snap

```diff
@@ -592,12 +573,10 @@ exports[`<Checkbox /> properly renders a disabled indeterminate checkbox 1`] = `
     disabled={true}
     id="example"
     name="example"
-    onChange={[Function]}
```

Comment:
this being removed doesn't seem right

Reply:
`defaultProps.onChange` was `() => {}`, so I left it undefined instead.  However, it turns out there's a warning when `checked` is passed with no `onChange`, so that probably explains it, and I'll put the default back.